app/stac_api/widgets/label_widget.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.utils.safestring import mark_safe
+
+
+class LabelWidget(forms.TextInput):
```

Comment:
I'm a bit confused about why this inherits `TextInput`. This should always display a text, shouldn't it? Or is there a case where `if value` isn't met?